examples/react-widgets/src/ReactWidgetsForm.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react'
+import { Field, reduxForm } from 'redux-form'
+import DropdownList from 'react-widgets/lib/DropdownList'
+import SelectList from 'react-widgets/lib/SelectList'
+import Multiselect from 'react-widgets/lib/Multiselect'
+import 'react-widgets/dist/css/react-widgets.css'
+import { connect } from 'react-redux'
+import { load as loadData } from './userData'
+
+const initialData = {
+  hobbies: [ 'Guitar', 'Hiking' ],
+  sex: 'female',
+  favoriteColor: 'Blue'
+}
+
+const colors = [ { color: 'Red', value:'ff0000' },
+  { color: 'Green', value:'00ff00' },
+  { color: 'Blue', value:'0000ff' } ]
+
+const validate = values => {
+  const errors = {}
+  const requiredFields = [ 'favoriteColor', 'hobbies', 'sex' ]
+  requiredFields.forEach(field => {
+    if (!values[ field ]) {
+      errors[ field ] = 'Required'
+    }
+  })
+  return errors
+}
+
+let ReactWidgetsForm = props => {
+  const { handleSubmit, pristine, reset, submitting, load } = props
+  return (
+    <form onSubmit={handleSubmit}>
+    <div>
+        <button type="button" onClick={() => load(initialData)}>Initilalize</button>
+    </div>
+    <div>
+      <label>Favorite Color</label>
+      <Field name="favoriteColor"
+        component={ favoriteColor =>
```

Comment:
@ooflorent  I wasn't aware of this API change. I will look around and modify it accordingly. Thanks for reviewing.